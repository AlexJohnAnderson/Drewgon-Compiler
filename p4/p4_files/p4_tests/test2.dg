int a;
int main(){
	int n;
	n = 5;
	a = n;
	return a;
}
