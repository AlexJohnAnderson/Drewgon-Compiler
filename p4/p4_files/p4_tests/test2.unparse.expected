int a(int);
int main(->int)(){
    int n(int);
    n(int) = 5;
    a(int) = n(int);
    return a(int);
}